### PR TITLE
115 privateimagehubs dockerfile

### PR DESCRIPTION
**Pull request by anneliawa**

### CODE REVIEW

File: pipeline-runner/steps/build_acr.go [hunk 1]
```diff
@@ -334,16 +338,28 @@ func getBuildAcrJobContainerVolumeMounts(azureServicePrincipleContext string, bu
...
 	commandList := commandbuilder.NewCommandList()
-
 	commandList.AddStrCmd("/usr/bin/buildah login --username ${BUILDAH_USERNAME} --password ${BUILDAH_PASSWORD} %s", containerImageRegistry)
 	if useBuildCache {
 		commandList.AddStrCmd("/usr/bin/buildah login --username ${BUILDAH_CACHE_USERNAME} --password ${BUILDAH_CACHE_PASSWORD} %s", cacheContainerImageRegistry)
 	}
-
+	for registry, auth := range additionalDockerAuth.Auths {
+		if len(auth.Username) > 0 && len(auth.Password) > 0 {
+			commandList.AddStrCmd("/usr/bin/buildah login --username %s --password %s %s", auth.Username, auth.Password, registry)
```
Comment (Richard87) [inline pipeline-runner/steps/build_acr.go:360]: Move password (and username?) to environment variables so they are not visible in the Pod definition

**Review by Richard87 — CHANGES_REQUESTED** [review]
Liten merknad

File: pipeline-runner/steps/build_acr.go (hunk 1 above)
Comment (nilsgstrabo) [reply pipeline-runner/steps/build_acr.go]: Fixed. The radix-private_image_hubs secrets is mounted in the build containers and used as authfile for buildah

**Review by Richard87 — APPROVED** [review]
Godkjent

**Pull request merged**